kratos/solving_strategies/convergence_accelerators/convergence_accelerator.h

```diff
@@ -173,6 +191,25 @@ class ConvergenceAccelerator
     // Level of echo for the convergence accelerator
     int mEchoLevel;
 
+    ///@}
+    ///@name Protected  Access
+    ///@{
+
+    virtual void ComputeJacobianApproximation()
+    {
+        KRATOS_ERROR << "Interface Jacobian approximation cannot be computed with this convergence accelerator." << std::endl;
+    }
+
+    virtual DenseMatrixPointerType pGetInverseJacobianApproximation()
+    {
+        KRATOS_ERROR << "Interface Jacobian approximation is not available for this convergence accelerator." << std::endl;
+    }
+
+    virtual unsigned int GetProblemSize()
```

Comment:
```suggestion
    virtual size_t GetProblemSize()
```

why don't you use `size_t`? I think it would be really unfortunate to run into overflow issues with large problems